editor.planx.uk/src/@planx/components/MapAndLabel/Editor.tsx

```diff
@@ -0,0 +1,80 @@
+import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
+import { useFormik } from "formik";
+import React from "react";
+import InputGroup from "ui/editor/InputGroup";
+import ModalSection from "ui/editor/ModalSection";
+import ModalSectionContent from "ui/editor/ModalSectionContent";
+import RichTextInput from "ui/editor/RichTextInput";
+import Input from "ui/shared/Input";
+import InputRow from "ui/shared/InputRow";
+
+import { EditorProps, ICONS, InternalNotes, MoreInformation } from "../ui";
+import { MapAndLabel, parseContent } from "./model";
+
+type Props = EditorProps<TYPES.MapAndLabel, MapAndLabel>;
+
+export default MapAndLabelComponent;
+
+function MapAndLabelComponent(props: Props) {
+  const formik = useFormik({
+    initialValues: parseContent(props.node?.data),
+    onSubmit: (newValues) => {
+      props.handleSubmit?.({
+        type: TYPES.MapAndLabel,
+        data: newValues,
+      });
+    },
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit} id="modal">
+      <ModalSection>
+        <ModalSectionContent
+          title="Map and label"
+          Icon={ICONS[TYPES.MapAndLabel]}
+        >
+          <InputGroup>
+            <InputRow>
+              <Input
+                format="large"
+                name="title"
+                placeholder={"Title"}
+                value={formik.values.title}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
+            <InputRow>
+              <RichTextInput
+                name="description"
+                placeholder="Description"
+                value={formik.values.description}
+                onChange={formik.handleChange}
+              />
+            </InputRow>
+            <InputRow>
+              <Input
+                format="data"
+                name="fn"
+                placeholder={"Data Field"}
+                value={formik.values.fn}
+                disabled
```

Comment:
Why is this one `disabled` ?

Reply:
Lingering mistake from me turning things off and on! I'll correct before merging